doc/en/user/source/styling/mbstyle/cookbook/lines.rst

```diff
@@ -0,0 +1,295 @@
+.. _mbstyle_cookbook.lines:
+
+Lines
+=====
+
+While lines can also seem to be simple shapes, having length but no width, there are many options and tricks for making
+lines display nicely.
+
+.. _mbstyle_cookbook_lines_attributes:
+
+Example lines layer
+-------------------
+
+The :download:`lines layer <artifacts/mbstyle_cookbook_line.zip>` used in the examples below contains road information for a
+fictional country. For reference, the attribute table for the points in this layer is included below.
+
+.. list-table::
+   :widths: 30 40 30
+   :header-rows: 1
+
+   * - ``fid`` (Feature ID)
+     - ``name`` (Road name)
+     - ``type`` (Road class)
+   * - line.1
+     - Latway
+     - highway
+   * - line.2
+     - Crescent Avenue
+     - secondary
+   * - line.3
+     - Forest Avenue
+     - secondary
+   * - line.4
+     - Longway
+     - highway
+   * - line.5
+     - Saxer Avenue
+     - secondary
+   * - line.6
+     - Ridge Avenue
+     - secondary
+   * - line.7
+     - Holly Lane
+     - local-road
+   * - line.8
+     - Mulberry Street
+     - local-road
+   * - line.9
+     - Nathan Lane
+     - local-road
+   * - line.10
+     - Central Street
+     - local-road
+   * - line.11
+     - Lois Lane
+     - local-road
+   * - line.12
+     - Rocky Road
+     - local-road
+   * - line.13
+     - Fleet Street
+     - local-road
+   * - line.14
+     - Diane Court
+     - local-road
+   * - line.15
+     - Cedar Trail
+     - local-road
+   * - line.16
+     - Victory Road
+     - local-road
+   * - line.17
+     - Highland Road
+     - local-road
+   * - line.18
+     - Easy Street
+     - local-road
+   * - line.19
+     - Hill Street
+     - local-road
+   * - line.20
+     - Country Road
+     - local-road
+   * - line.21
+     - Main Street
+     - local-road
+   * - line.22
+     - Jani Lane
+     - local-road
+   * - line.23
+     - Shinbone Alley
+     - local-road
+   * - line.24
+     - State Street
+     - local-road
+   * - line.25
+     - River Road
+     - local-road
+
+:download:`Download the lines shapefile <artifacts/ysld_cookbook_line.zip>`
+
+.. _mbstyle_cookbook_lines_simpleline:
+
+Simple line
+-----------
+
+This example specifies lines be colored black with a thickness of 3 pixels.
+
+.. figure:: ../../sld/cookbook/images/line_simpleline.png
+
+   Simple line
+
+Code
+~~~~
+
+:download:`Download the "Simple line" MBStyle <artifacts/mbstyle_line_simpleline.json>`
+
+.. code-block:: json
+  :linenos:
+
+  {
+    "version": 8,
+    "name": "simple-line",
+    "layers":[
+      {
+        "id": "simple-line",
+        "type": "line",
+        "paint": {
+          "line-color": "#000000",
+          "line-width": "3"
+            }
+          }
+        ]
+    }
+
+Details
+~~~~~~~
+
+There is one layer style for this MBStyle, which is the simplest possible situation. Styling
+lines is accomplished in the paint (**lines 8-10**). **Line 9** specifies the color of the line to be

```

Comment:
Not quite accurate, as lines can also be styled in the layout section (although that is not used in this example). 

Perhaps use "Styling lines is accomplished using the line layer" instead?